### PR TITLE
Make rpm-spec-merge-driver usable as a standalone tool

### PR DESCRIPTION
**Pull request by encukou**

### CODE REVIEW

**Comment by encukou** [comment]
@hroncok, do you want to take a look?

**Comment by hroncok** [comment]
Sorry, I lost track. Sure, but not sure if today.

**Comment by encukou** [comment]
OK.

File: tests/test_cases.py [hunk 1]
```diff
@@ -36,6 +38,19 @@ def run(*argv, **kwargs):
...
 
+def init_git(repo_path):
+    run('git', 'init', cwd=repo_path)
+
+    run('git', 'config', 'merge.rpm-spec.name', 'RPM spec file merge driver', cwd=repo_path)
+    run('git', 'config', 'merge.rpm-spec.driver', f'{TOOL} %O %A %B %L %P', cwd=repo_path)
+    run('git', 'config', 'merge.rpm-spec.recursive', 'text', cwd=repo_path)
+    run('git', 'config', 'merge.verbosity', '5', cwd=repo_path)
+    run('git', 'config', 'merge.conflictStyle', 'diff3', cwd=repo_path)
+    run('git', 'config', 'user.name', 'Merge Driver User', cwd=repo_path)
```
Comment (hroncok) [inline tests/test_cases.py:49]: Curious question: 💁 was removed from here. Is that on purpose?

File: tests/test_cases.py [hunk 2]
```diff
@@ -49,16 +64,7 @@ def test_case(case_filename, tmp_path):
...
     attrs_path = repo_path / '.gitattributes'
 
-    run('git', 'init', cwd=repo_path)
-
-    run('git', 'config', 'merge.rpm-spec.name', 'RPM spec file merge driver', cwd=repo_path)
-    run('git', 'config', 'merge.rpm-spec.driver', f'{TOOL} %O %A %B %L %P', cwd=repo_path)
-    run('git', 'config', 'merge.rpm-spec.recursive', 'text', cwd=repo_path)
-    run('git', 'config', 'merge.verbosity', '5', cwd=repo_path)
-    run('git', 'config', 'merge.conflictStyle', 'diff3', cwd=repo_path)
-    run('git', 'config', 'user.name', 'Merge Driver 💁', cwd=repo_path)
```
Comment (hroncok) [inline tests/test_cases.py:59]: ..from here.

**Review by hroncok — APPROVED** [review]
The code looks good. One curious question.

Suggestion: Add example to the README?

PS Sorry for the delay.

File: tests/test_cases.py (hunk 1 above)
Comment (encukou) [reply tests/test_cases.py:49]: Since #8, the name here doesn't matter. The changed name doesn't appear in actual tests.

**Comment by encukou** [comment]
Thanks for the review!

**Pull request merged**